Chiazokam/Politico
# #163498861 create the create candidate endpoint

#### What does this PR do?
This PR tests and creates the create candidate Endpont

#### Description of Task to be completed?
- write tests for every test case
- integrate admin check middleware
- create the create candidate endpoint
- implement all feedback from LFA

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-create-candidate-163498861`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/offices/1/register`
- You should receive a successful response

#### Any background context you want to provide?
`
{
  "office": "3",
  "party": "8"
}
`

#### What are the relevant pivotal tracker stories?
[#163498861](https://www.pivotaltracker.com/story/show/163498861)

### Images
![image](https://user-images.githubusercontent.com/26940294/51982170-80bb5b00-2495-11e9-97d9-9ed1fe0dd643.png)
